Classes/Emogrifier.php

```diff
@@ -262,6 +262,10 @@ public function emogrify()
             // query the body for the xpath selector
             $nodesMatchingCssSelectors = $xpath->query($this->translateCssToXpath($value['selector']));
 
+            // Ignore invalid selectors
+            if (!$nodesMatchingCssSelectors) {
```

Comment:
This is not always a boolean. So we should not treat it as a boolean, but do an explicit === false instead.